app/views/disbursements/_form.html.erb

```diff
@@ -1,57 +1,10 @@
 
-  <div class="field event-select-target">
-    <%= form.label :source_event_id, "From" %>
-    <%= form.collection_select(:source_event_id,
-                                @allowed_source_events,
-                                :id,
-                                current_user.admin? ? :admin_dropdown_description : :disbursement_dropdown_description,
-                                { prompt: "Select one…", disabled: ->(e) { e != @source_event && !policy(e).create_transfer? || (!current_user.admin? && e.balance_available <= 0) } },
-                                required: true,
-                                disabled: ) %>
-  </div>
-
-  <div class="field event-select-target">
-    <%= form.label :destination_event_id_select, "To" %>
-    <%= form.select(:event_id,
-                  (@allowed_destination_events.map do |event|
-                    [
-                      current_user.admin? ? event.admin_dropdown_description : event.disbursement_dropdown_description,
-                      event.id,
-                      {
-                        "data-fee": event.revenue_fee.to_s,
-                        "data-event_id": event.id
-                      },
-                    ]
-                  end) +
-                  (@source_event&.plan&.unrestricted_disbursements_enabled? && !admin_signed_in? ? [[
-                    "Other",
-                    "other",
-                    {
-                      "data-fee": "null",
-                      "data-event_id": "other"
-                    },
-                  ]] : []),
-                  { prompt: "Select one…" },
-                  required: true,
-                  disabled:,
-                  "x-on:input" => "fee = $event.target.options[$event.target.selectedIndex].dataset.fee; event_id = $event.target.options[$event.target.selectedIndex].dataset.event_id; console.log(event_id)") %>
-    <% unless admin_signed_in? %>
-      <% if @source_event&.plan&.unrestricted_disbursements_enabled? %>
-        <template x-if="event_id == 'other'">
-          <div class="mt2">
-            <%= form.label :destination_event_id_select, "Destination event's slug" %>
-            <%= form.text_field :event_id, placeholder: "hack_the_seas", required: true, disabled:, data: { behavior: "extract_slug" } %>
-          </div>
-        </template>
-      <% else %>
-        <span class="muted">You can transfer to any organization you're a part of.</span>
-      <% end %>
-    <% end %>
-  </div>
+  <%= render partial: "select_organization", locals: { form: form, field_name: "source_event_id", disabled: disabled, events: @allowed_source_events, default_event: @source_event, sending: true } %>
+  <%= render partial: "select_organization", locals: { form: form, field_name: "event_id", disabled: disabled, events: @allowed_destination_events, default_event: nil, hide_balances: @source_event&.plan&.unrestricted_disbursements_enabled?, receiving: true } %>
```

Comment:
Imo the partial should be `events/_select.html.erb`

Reply:
Oh I guess we aren't planning to make this a general purpose organisations selector. In that case just `select_event`